models/enterprise_sonic/interfaces/replaced_example_01.txt

```diff
@@ -5,25 +5,36 @@
 #
  show interface status | no-more
 ------------------------------------------------------------------------------------------
-Name                Description         Admin          Oper           Speed          MTU
+Name                Description         Admin     Oper      AutoNeg     Speed        MTU
 ------------------------------------------------------------------------------------------
-Ethernet0           -                   up                            100000         9100
-Ethernet4           -                   up                            100000         9100
-Ethernet8           -                   down                          100000         9100
-Ethernet12          -                   down                          1000           9100
-Ethernet16          -                   down                          100000         9100
+Ethernet0           -                   up                              100000       9100
+Ethernet4           -                   up                              100000       9100
+Ethernet8           -                   down                            100000       9100
+Ethernet12          -                   down                            1000         9100
+Ethernet16          -                   down                            100000       9100
+
+ show running-configuration interface Ethernet 8
```

Comment:
In this file, also, the "before" and "after" output of the "show running-configuration interface" command for all three interfaces (Ethernet8, Ethernet12, and Ethernet 16) would help to show how this "state" operates.

Reply:
For the current version of this example, the Ethernet12 and Ethernet16 "show running-configuration" output would not provide additional useful information. (See the related discussion for the other examples.)

For that reason, this issue can be marked "Resolved".

Note: A new issue is being added to request a clearer illustration of how the "replaced" state operates for this resource module.